data-raw/process_data_sets_kara.R

```diff
@@ -0,0 +1,35 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#reading in csv
+nba_elo <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo.csv")
+nba_elo_latest <- read_csv("https://projects.fivethirtyeight.com/nba-model/nba_elo_latest.csv")
+
+rownames(nba_elo) <- NULL
+rownames(nba_elo_latest) <- NULL
+
+#tidying and taming data for nba_elo
+nba_elo <- nba_elo %>% clean_names()
+
+nba_elo <- nba_elo %>%
+  mutate_if(is.logical, as.numeric) %>%
+  mutate(neutral = as.logical(neutral)) %>%
+  mutate(playoff = as.logical(playoff))
+
+
+#tidying and taming data for nba_elo_latest
+nba_elo_latest <- nba_elo_latest %>% clean_names()
```

Comment:
Could you combine lines 24 thru 29 into a single pipe chain, with one operation per line?